tests/bluetooth/tester/src/audio/btp_bap_broadcast.h

```diff
@@ -41,24 +41,32 @@ struct btp_bap_broadcast_remote_source {
 };
 
 struct btp_bap_broadcast_local_source {
+	bool allocated;
+	uint8_t source_id;
+	struct bt_le_ext_adv *ext_adv;
 	uint32_t broadcast_id;
 	struct bt_bap_qos_cfg qos;
 	struct btp_bap_broadcast_stream streams[CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT];
 	struct bt_audio_codec_cfg subgroup_codec_cfg[CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT];
 	uint8_t stream_count;
+	struct bt_data per_adv_local[10];
```

Comment:
Why does this need to be `10`? Don't we just need 1 for the BASE? It's only ever accessed as `per_adv_local[0]`